src/components/RadioButtons/index.tsx

```diff
@@ -0,0 +1,69 @@
+import 'tailwindcss/tailwind.css'
+import { FC, useState } from 'react'
+import { VARIANT, VARIANT_STATES_RADIO_BUTTONS } from '@/types/type'
+
+type ClassName = Record<
+  VARIANT,
+  Record<VARIANT_STATES_RADIO_BUTTONS, string> | Record<string, never>
+>
+
+export const className: ClassName = {
+  [VARIANT.PRIMARY]: {
+    [VARIANT_STATES_RADIO_BUTTONS.NOT_CHECKED]:
+      'appearance-none w-6 h-6 rounded-full border-2 border-primary-base  cursor-pointer ',
+    [VARIANT_STATES_RADIO_BUTTONS.CHECKED]:
+      'bg-primary-base w-4 h-4 rounded-full block absolute left-1 top-1',
+  },
+  [VARIANT.SECONDARY]: {
+    [VARIANT_STATES_RADIO_BUTTONS.NOT_CHECKED]:
+      'appearance-none w-6 h-6 rounded-full border-2 border-secondary-base  cursor-pointer',
+    [VARIANT_STATES_RADIO_BUTTONS.CHECKED]:
+      ' bg-secondary-base w-4 h-4 rounded-full block absolute left-1 top-1',
+  },
+  [VARIANT.NEUTRAL]: {
+    [VARIANT_STATES_RADIO_BUTTONS.NOT_CHECKED]:
+      'appearance-none w-6 h-6 rounded-full border-2 border-nautral-base  cursor-pointer ',
```

Comment:
en el not-checked te falta la lógica para pintar el relleno atenuado cuando se hace hover